output/transform/postgres.go

```diff
@@ -59,7 +60,7 @@ func transformPostgresRoles(s snapshot.FullSnapshot, transientState state.Transi
 	return s, roleOidToIdx
 }
 
-func transformPostgresDatabases(s snapshot.FullSnapshot, transientState state.TransientState, roleOidToIdx OidToIdx) (snapshot.FullSnapshot, OidToIdx) {
+func transformPostgresDatabases(s snapshot.FullSnapshot, newState state.PersistedState, diffState state.DiffState, transientState state.TransientState, roleOidToIdx OidToIdx) (snapshot.FullSnapshot, OidToIdx) {
```

Comment:
I don't think the newState argument is used anymore (probably a leftover from an earlier version of the code)